topics/icrc_34_delegation.md

```diff
@@ -30,20 +32,24 @@ The delegation chain's stable identifier can be either:
 stable across many relying parties but cannot be used to operate on tradable assets and shared infrastructure.
 2. **Relying Party Delegation**: an identity created exclusively for the relying party (Relying Party Delegation).
 
-Example use cases of a Relying Party Delegation:
-- Signer could give the user a choice to stay anonymous with a service.
-- Differentiate between calls made with user approval (Account identifier) and without user approval (Relying Party Delegation identifier), allowing for fine grained security levels per identifier.
-- Exclusive identifier within the Relying Party platform to stay isolated from identifiers of other Relying Party platforms.
+Relying Party Delegations **MUST** be for identities exclusive to each individual relying party. This property **MUST** be enforced by the signer.
+Signers **MAY** give users the choice to authenticate with their relying party specific identifier instead of accounts (if available).
 
 If a relying party wants to receive an Account Delegation, the `icrc34_delegation` request **MUST** include canisters it
-controls as `targets` where each canister **MUST** implement the `icrc28_trusted_origins` endpoint as per the 
+controls as `targets` where each canister **MUST** implement the `icrc28_trusted_origins` endpoint as per the
 [ICRC-28](./icrc_28_trusted_origins.md) standard.
+Relying parties must not include `targets` in the request if they want to be guaranteed to receive a Relying Party delegation.
 
-Relying Party Delegations **MUST** be for identities exclusive to each individual relying party. This property **MUST** be enforced by the signer.
+### Use-Cases for Account Delegation:
+- Single identifier for the user that is the same for all relying parties and shared infrastructure, which leads to the following benefits:
+  - application specific user data can be directly tied to the users global identity without the need for a mapping to an application specific identifier
+  - easier integration and composition of applications that handle user data
 
-Signers **MAY** give users the choice to authenticate with their relying party specific identifier instead of accounts (if available).
+> **Note:** Using the same identifier globally may be detrimental to privacy as user activity can be tracked across applications.
 
-Relying parties must not include `targets` in the request if they want to be guaranteed to receive a Relying Party delegation.
+### Use-Cases for Relying Party Delegation:
+- Differentiate between calls made with user approval (Account identifier) and without user approval (Relying Party Delegation identifier), allowing for fine grained security levels per identifier.
+- Exclusive identifier within the Relying Party platform to stay isolated from identifiers of other Relying Party platforms. Depending on the signer implementation, this may offer privacy benefits.
 
```

Comment:
Let's add a note to the relying party delegation use case to give more context on the trade-off:

> **Note:** Using exclusive identifiers within the Relying Party platform may be detrimental to data portability, dapp integration, and composition of applications as user activity can not be easily queried across the ecosystem of dapps and services.

Reply:
Added.